docs/getting-started-logging.md

```diff
@@ -0,0 +1,75 @@
+| | +--------------+  +--------------+ | ---------------------(Logging)------> | Scalar Products | |
+| | |     Loki     |  |   Promtail   | |                                       |                 | |
+| | +--------------+  +--------------+ |                                       |  +-----------+  | |
+| +------------------------------------+                                       |  | Scalar DB |  | |
+|                                                                              |  +-----------+  | |
+| +------------------------------------------------------+                     |                 | |
+| |                kube-prometheus-stack                 |                     |  +-----------+  | |
+| |                                                      |                     |  | Scalar DL |  | |
+| | +--------------+  +--------------+  +--------------+ | ---(Monitoring)---> |  +-----------+  | |
+| | |  Prometheus  |  | Alertmanager |  |   Grafana    | |                     +-----------------+ |
+| | +-------+------+  +------+-------+  +------+-------+ |                                         |
+| |         |                |                 |         |                                         |
+| |         +----------------+-----------------+         |                                         |
+| |                          |                           |                                         |
+| +--------------------------+---------------------------+                                         |
+|                            |                                                                     |
+|                            |         Kubernetes (minikube)                                       |
++----------------------------+---------------------------------------------------------------------+
+                             | <- expose to localhost (127.0.0.1)
+                             |
+              (Access Dashboard through HTTP)
+                             |
+                        +----+----+
+                        | Browser |
+                        +---------+
+```
+
+## Step 1. Prepare a custom values file
+
+1. Get the sample file [scalar-loki-stack-custom-values.yaml](./conf/scalar-loki-stack-custom-values.yaml) for `loki-stack`.
+
+## Step 2. Deploy `loki-stack`
+
+1. Add the `grafana` helm repository.
+   ```console
+   helm repo add grafana https://grafana.github.io/helm-charts
+   ```
+
+1. Deploy the `loki-stack`.
+   ```console
+   helm install scalar-logging-loki grafana/loki-stack -n monitoring -f scalar-loki-stack-custom-values.yaml
+   ```
+
+## Step 3. Access Dashboards
+
+1. Access log browser
+   - Go to Grafana http://localhost:3000
+   - Move to `Configuration` and choose `Data Sources`
+   - Click `Add data source`
+   - Select `Loki`
+   - Input `http://scalar-logging-loki:3100` to URL
+   - Click `Save and test`
+   - Go to `Explore` to find the added Loki
+
+## Step 4. Delete `loki-stack`
```

Comment:
Ditto.

Reply:
Is it ok to keep `Delete loki-stack` to be consistent with
```
## Step 2. Deploy `loki-stack`
```
?

Reply:
I meant
```suggestion
## Step 4. Delete the `loki-stack` helm chart
```
for consistency.